packages/domainproperties/src/components/assay/AssayDesignerPanels.tsx

```diff
@@ -181,48 +247,60 @@ export class AssayDesignerPanels extends React.PureComponent<Props, State> {
+                            panelStatus={protocolModel.isNew() ? this.getPanelStatus(i + 1) : "COMPLETE"}
                             showInferFromFile={showInferFromFile}
                             containerTop={containerTop}
-                            helpURL={null} // don't show the helpURL link for assay domains since we have a help link in the Assay Properties panel
-                            onChange={(updatedDomain) => {
+                            helpURL={null} // so we only show the helpURL link for the first assay domain
+                            onChange={(updatedDomain, dirty) => {
                                 this.onDomainChange(i, updatedDomain);
                             }}
+                            onToggle={(collapsed, callback) => {
+                                this.onTogglePanel((i + 1), collapsed, callback);
+                            }}
                             appDomainHeaderRenderer={appDomainHeaderRenderer}
                             modelDomains={protocolModel.domains}
+                            useTheme={useTheme}
                         >
                             <div>{domain.description}</div>
                         </DomainForm>
                     )
                 })}
-                {error && <Alert>{error}</Alert>}
-                {transformScriptError && <Alert>{transformScriptError}</Alert>}
-                <WizardNavButtons
-                    cancel={onCancel}
-                    canFinish={this.isValid()}  //if finished call isValid, otherwise isNameSet
-                    canNextStep={protocolModel.isValid()}
-                    canPreviousStep={!submitting && currentPanelIndex > 0}
-                    containerClassName=""
-                    finish={finish}
-                    includeNext={!finish}
-                    isFinishing={submitting}
-                    nextStep={finish ? this.onFinish : this.onNext}
-                    nextStyle={'success'}
-                    previousStep={isNew ? this.onPrevious : undefined}
-                />
+                <Row className='domain-field-padding-top'>
+                    <Col xs={1}>
+                        <Button className='domain-assay-save-btn' onClick={onCancel}>Cancel</Button>
+                    </Col>
+                    <Col xs={10}>
+                        {errorDomains.size > 0 &&
+                            <Alert bsStyle="danger">{"Must correct errors in " + errorDomains.join(', ') + " before saving."}</Alert>
+                        }
+                        {visitedPanels.size > 1 && !protocolModel.hasValidProperties() &&
+                            <Alert bsStyle="danger">Must correct errors in Assay Properties before saving.</Alert>
+                        }
+                        {protocolModel.exception &&
+                        <Alert bsStyle="danger">{protocolModel.exception}</Alert>
+                        }
+                    </Col>
```

Comment:
We haven't been putting the error message inbetween the buttons for the other use cases like this in the various app pages and single domain designer case. Was this discussed with MattY? For consistency I was expecting to see this message show between the last panel and the buttons.